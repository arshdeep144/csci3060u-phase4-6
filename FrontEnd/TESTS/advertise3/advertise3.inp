login
user01
advertise
2011 335i xdrive bmw with leather seats functioning turning lights
999.99
90
logout
