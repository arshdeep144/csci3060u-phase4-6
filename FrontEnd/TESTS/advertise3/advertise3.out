Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
You dont have access to this feature with your current account role
